### PR TITLE
Add command for schema class generation

### PR DESCRIPTION
**Pull request by matsn0w**
## Summary
- Added a new command: `php artisan make:graphql:schema <name>`
- Added test for new command

This new command can be used to quickly generate a new schema configuration class.

Type of change:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Misc. change (internal, infrastructure, maintenance, etc.)

Checklist:
- [x] Existing tests have been adapted and/or new tests have been added
- [x] Add a CHANGELOG.md entry
- [x] Update the README.md
- [x] Code style has been fixed via `composer fix-style`

### CODE REVIEW

File: src/Console/SchemaMakeCommand.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        return $this->replaceGraphqlName($stub);
+    }
+
+    protected function replaceGraphqlName(string $stub): string
+    {
+        $graphqlName = lcfirst($this->getNameInput());
+        $graphqlName = preg_replace('/Schema$/', '', $graphqlName);
+
+        return str_replace(
+            'DummyGraphqlName',
```
Comment (mfn) [inline src/Console/SchemaMakeCommand.php:37]: Shouldn't this match the class `DummyClass`? 🤔

**Review by mfn — CHANGES_REQUESTED** [review]
LGTM!

But I think the command (and this all the class / test names, etc.) should be named `schemaConfiguration` (also fine with `schemaConfig`).

Otherwise it might be confusing because a "graphql schema" is definitely something else.

WDYT?

File: src/Console/SchemaMakeCommand.php (hunk 1 above)
Comment (matsn0w) [reply src/Console/SchemaMakeCommand.php:37]: oh lol, that entire method is a copy/paste artifact XD

will fix in a minute

Comment (matsn0w) [reply src/Console/SchemaMakeCommand.php]: see https://github.com/rebing/graphql-laravel/pull/830/commits/e3e905e080103d98207c3f0b40e269ddc22e9313

**Comment by matsn0w** [comment]
> But I think the command (and this all the class / test names, etc.) should be named `schemaConfiguration` (also fine with `schemaConfig`).
> 
> Otherwise it might be confusing because a "graphql schema" is definitely something else.
> 
> WDYT?

That makes sense. I'd go with the shorter `schemaConfig` for the sake of not having to type that out entirely ;)

**Review by mfn — APPROVED** [review]
LGTM!

---
However I've to merge your PR via https://github.com/rebing/graphql-laravel/pull/840 because you had PHP 8 specific changes (the project is PHP 7.2+) and some other phpstan issue I solved there.

Rest assured your authorship is retained!

**Pull request merged**